dcs/mapping.py

```diff
@@ -97,6 +97,11 @@ class Vector2:
     x: float
     y: float
 
+@dataclass
+class WWIIFollowBigFormationOrder:
```

Comment:
This should probably go in `task.py`. It's got nothing to do with mapping afaict.